Modules/Devices/src/Devices.Infrastructure/Persistence/Repository/OAuthClientsRepository.cs

```diff
@@ -45,7 +45,7 @@ public async Task<OAuthClient> Find(string clientId, CancellationToken cancellat
         if (track)
             Track(application);
 
-        return application?.ToModel();
+        return application?.ToModel()!;
```

Comment:
Not quite sure if this should be suppressed though.

Reply:
No it should not. As in nearly all repositories, the return value of the `Find` methods should be nullable

Reply:
I mean... think about it. The `application` is nullable (=> it can be null). And the `?` makes the expression return null in case `application` is null. So why would you lie in the signature of the `OAuthClientsRepository` by saying that the method NEVER returns null?

Reply:
@NikolaDmitrasinovic ;)

Reply:
Yeah, makes sense.